### PR TITLE
Set up GRO Drone CI

### PR DESCRIPTION
**Pull request by JoeChapman**
- remove travis
- add drone config
- sign drone config and generate signature
- amend Dockerfile
- use Dockerfile acceptance to run acc tests in a container
- update gems
- display a drone build status badge in the readme

### CODE REVIEW

**Comment by joefitter** [comment]
ja ja ja looks lovely

**Comment by easternbloc** [comment]
need to disable the travis build in settings of the repo. Also need to pass CI muhaahaha

**Comment by JoeChapman** [comment]
Yeah, I have disabled travis, maybe just waiting for another trigger.
Drone CI is not passing because "Warning: unable to verify the Yaml signature."

**Comment by JoeChapman** [comment]
@joefitter @easternbloc please can this be merged?

File: .gitignore [hunk 1]
```diff
@@ -2,7 +2,6 @@ logs
 *.log
 node_modules
 .idea/
-Gemfile.lock
```
Comment (joefitter) [inline .gitignore]: L8RS

**Comment by joefitter** [comment]
LGTM @easternbloc ?

**Pull request merged**